packages/salesforcedx-vscode-core/test/vscode-integration/util/orgUtil.test.ts

```diff
@@ -27,115 +27,131 @@ describe('orgUtil tests', () => {
+        {
+          isDevHub: true,
+          username: 'bar',
+          aliases: [orgName2]
+        }
+      ]);
       const orgList = new OrgList();
       await checkForExpiredOrgs(orgList);
 
-      // Todo: fix spy
       expect(showWarningMessageSpy.called).to.equal(false);
       expect(appendLineSpy.called).to.equal(false);
       expect(showChannelOutputSpy.called).to.equal(false);
+      expect(authInfoCreateStub.called).to.equal(false);
     });
 
-    xit('should not display a notification when the scratch org has already expired', async () => {
-      // Todo: stub listAllAuthorizations
+    it('should not display a notification when the scratch org has already expired', async () => {
+      listAllAuthorizationsStub.resolves([
+        {
+          isDevHub: false,
+          username: 'foo',
+          aliases: [orgName1]
+        }
+      ]);
+
+      authInfoCreateStub.resolves({
+        getFields: () => {
+          return {
+            expirationDate: `${yesterday.getFullYear()}-${yesterday.getMonth() +
+              1}-${yesterday.getDate()}`
+          };
+        }
+      });
       const orgList = new OrgList();
       await checkForExpiredOrgs(orgList);
 
-      // Todo: fix spy
       expect(showWarningMessageSpy.called).to.equal(false);
       expect(appendLineSpy.called).to.equal(false);
       expect(showChannelOutputSpy.called).to.equal(false);
+      expect(authInfoCreateStub.called).to.equal(true);
     });
 
-    xit('should display a notification when the scratch org is about to expire', async () => {
+    it('should display a notification when the scratch org is about to expire', async () => {
       const orgName = 'dreamhouse-org';
-      const listAllAuthorizationsStub = sb
-        .stub(AuthInfo, 'listAllAuthorizations')
-        .resolves([
-          {
-            isDevHub: false,
-            username: 'foo',
-            aliases: [orgName]
-          }
-        ]);
-
-      const authInfoCreateStub = sb.stub(AuthInfo, 'create').resolves({
+      listAllAuthorizationsStub.resolves([
+        {
+          isDevHub: false,
+          username: 'foo',
+          aliases: [orgName]
+        }
+      ]);
+
+      authInfoCreateStub.resolves({
         getFields: () => {
           return {
-            expirationDate: `${today.getFullYear()}-${today.getMonth() +
-              1}-${today.getDate() + 3}`
+            expirationDate: `${threeDaysFromNow.getFullYear()}-${threeDaysFromNow.getMonth() +
+              1}-${threeDaysFromNow.getDate()}`
           };
         }
       });
 
       const orgList = new OrgList();
       await checkForExpiredOrgs(orgList);
 
-      // Todo: fix spy
       expect(showWarningMessageSpy.called).to.equal(true);
       expect(appendLineSpy.called).to.equal(true);
       expect(appendLineSpy.args[0][0]).to.contain(orgName);
       expect(showChannelOutputSpy.called).to.equal(true);
-
-      authInfoCreateStub.restore();
-      listAllAuthorizationsStub.restore();
```

Comment:
covered by the sb.restore in the afterEach

Reply:
restore() is fantastic